test/KHI_growthRate/bin/ci.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+currentPath=$(cd `dirname $0` && pwd)
+create="true"
+
+while [[ $# > 0 ]] ; do
+        case "$1" in
+
+                -t|--testPar)
+                        testPar="$2"
+                        shift
+                        ;;
+
+                -s|--simDir)
+                        simDir="$2"
+                        shift
+                        ;;
+
+                -f|--force)
+                        force="true"
+                        ;;
+
+                -c|--create)
+                        create="false"
+                        ;;
+
+                --help|-h)
+                        echo "If the parameter -t is not set, it is assumed that"
+                        echo " picongpu is in the home directory"
+                        echo "Usage:"
+                        echo "  --testPar|-t \"direction/to/picongpu/test/KHI_growthRate/\""
+                        echo "  --simDir|-s \"direction/where/simulation/should/be/build/\""
+                        echo "  -c|--create do not run pic-create"
+                        echo "  -f|--force  - Override if 'destinationPath' exists."
+                        echo "  --help|-h"
+                        echo "Example:"
+                        echo " bash ci.sh - creates a new temporary folder in which the "
+                        echo "              test-suite is run and then deletes it"
+                        exit 0
+                        ;;
+        esac
+        shift
+done
+
+if [ -z "$simDir" ] ; then
+        date_stamp=$(date +"%F-%h-%M-%S")
+        simDir=/tmp/$date_stamp
+fi
+
+if [ -d "$simDir" ] ; then
+    if [ "$force" == "true" ] ; then
+        echo "Warning: using existing folder on user-request [-f]"
+    else
+        echo "Destination path already in use, cannot create new folder" >&2
+        exit 1
+    fi
+fi
+
+if [ -z "$testPar" ] ; then
+        testPar=$HOME
+fi
+
+source etc/picongpu/bash-devServer-hzdr/gpu_picongpu.profile.example
+
+if [ "$create" == "true" ] ; then
+        mktemp -d $date_stamp
+        pic-create $testPar/picongpu/test/KHI_growthRate/ $simDir
+        currentPath=$simDir/bin/
+    else
+        mkdir -p $simDir
+fi
+
+cd $currentPath/../
+pic-build -b cuda:70
+
+mkdir $simDir/simOutput
+if [ ! -d "$simDir/include/picongpu/param/" ] ; then
```

Comment:
Why is this required?
I do not understand why we need to copy the include files after PIConGPU is compiled in line 74.

Reply:
Because I didn't want to pass another parameter to validate.sh. The copying can be removed if validate.sh gets another optional parameter giving it direction to .param files. With default simDir if not set. If this solution is better I'll change it.